### PR TITLE
Add roadmap section to the README

### PR DESCRIPTION
**Pull request by georg-schwarz**
We should share our roadmap and vision of Jayvee with the world.

See the rendered README [here](https://github.com/jvalue/jayvee/tree/document-roadmap)

## Screenshot

image

### CODE REVIEW

**Review by joluj — APPROVED** [review]

**Comment by rhazn** [comment]
I notice you always link to the rendered md file, did you know you can see the rich diff in files (https://github.com/jvalue/jayvee/pull/510/files) and there for all md files on the right side of the file name? It's a cool feature of github imho :D

**Review by rhazn — APPROVED** [review]
I am not a big fan of having the roadmap here because I assume it will get out of sync with what we are doing quick and that looks even worse (because now we have three different locations for a roadmap, the readme, the issues here, our internal tracking).

I would argue we should try to somehow have a public roadmap that auto updates itself. I realize that is hard though.

To not let perfect be the enemy of good, I am happy with approving this.

**Comment by georg-schwarz** [comment]
> I notice you always link to the rendered md file, did you know you can see the rich diff in files (https://github.com/jvalue/jayvee/pull/510/files) and there for all md files on the right side of the file name? It's a cool feature of github imho :D

Amazing :) I'm surprised that I successfully ignored that feature until now. Thanks! 

> I am not a big fan of having the roadmap here because I assume it will get out of sync with what we are doing quick and that looks even worse (because now we have three different locations for a roadmap, the readme, the issues here, our internal tracking).

I totally agree... I'd keep it on a very broad scope here so that it does not require frequent updates.

File: README.md [hunk 1]
```diff
@@ -12,6 +12,34 @@ Data engineers can use Jayvee and its interpreter to clean and preprocess data f
...
 
+## Roadmap
+
+Explore a glimpse of our upcoming features in the following list. This overview is broad and subject to evolution. We're excited to share our vision of the exciting journey ahead, and we invite you to accompany us on this adventure!
+
+- &#9989; Blocks and pipes 
+- &#9989; Simple valuetypes and constraints
+- &#9989; Natively support table-based data
+- &#9989; Column-based transformations
+- &#9989; Describe blocks via blocktypes in Jayvee
```
Comment (georg-schwarz) [inline README.md:23]:
```suggestion
- &#9989; Describe blocks via builtin blocktypes in Jayvee
- &#9989; Compose logic of multiple blocks via composite blocktypes
```

**Pull request merged**